urla958522fc1_spirosikmd_inmdsy_08_tgz-pJ8-TesterAppJ8/src/ft2server.java

```diff
@@ -0,0 +1,65 @@
+import java.io.*;
+import java.net.*;
+public class ft2server {
+
+	public static void main(String[] args) throws IOException {
+		
+		ServerSocket ss=null;
+        try
+        {  
+            ss=new ServerSocket(8081);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

**Problem**
This line of code contains a resource that might not be closed properly. This can cause a resource leak that slows down or crashes your system.

**Fix**
Consider closing the following resource: *ss*. The resource is referenced at line 20. The resource is closed at lines 51, 57. There are other execution paths that do not contain closure statements, for example, when ServerSocket.accept throws an exception. Ensure *ss* is closed in a try-finally block or declared in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).